guest_api/__main__.py

```diff
@@ -71,6 +71,7 @@ async def repost(request: web.Request):
 
 async def properties(request: web.Request):
     logger.debug("Forwarding signing properties")
+    _ = request
```

Comment:
Some tool is complaining about unused variables?

Reply:
Exactly